doc/rrdgraph.pod

```diff
@@ -115,13 +115,15 @@ the graph.
 [B<-u>|B<--upper-limit> I<value>]
 [B<-l>|B<--lower-limit> I<value>]
 [B<-r>|B<--rigid>]
+[B<--allow-shrink>]
 
 By default the graph will be autoscaling so that it will adjust the
 y-axis to the range of the data. You can change this behavior by
 explicitly setting the limits. The displayed y-axis will then range at
 least from B<lower-limit> to B<upper-limit>. Autoscaling will still
 permit those boundaries to be stretched unless the B<rigid> option is
-set.
+set. B<allow-shrink> alert B<rigid> behavior by allowing auto down scale,
```

Comment:
the word 'alert' seems odd in this sentence

Reply:
it was typo, it should be "alters", fixed